packages/team-management/src/components/TeamsView/Table.tsx

```diff
@@ -64,7 +64,18 @@ const Table: React.FC<Props> = (props: Props) => {
     },
   ];
 
-  return <AntTable dataSource={props.data} columns={columns} />;
+  return (
+    <AntTable
```

Comment:
should we add a test for this?

Reply:
These are the default props from AntD for Table component,
I dont think we need to add any tests for AntD default components.

Reply:
I mean do we need a test that shows the pagination was rendered correctly on the page, to guard against regressions.

Reply:
hmm, not sure how can I handle these changes to cover tests, or can I ? As we dont have any clicks or callback functions for this.

Reply:
the goal is just to see if in the rendered output there is sth that suggests the pagination is in the page
.